app/bundles/LeadBundle/Entity/LeadEventLogRepository.php

```diff
@@ -110,10 +110,10 @@ public function getEventsByLead($bundle, $object, Lead $lead, array $options)
             ->setParameter('object', $object);
 
         if (isset($options['search']) && $options['search']) {
-            $qb->andWhere($qb->expr()->like($alias.'.original_file', $qb->expr()->literal('%'.$options['search'].'%')));
+            $qb->andWhere($qb->expr()->like($alias.'.action', $qb->expr()->literal('%'.$options['search'].'%')));
```

Comment:
I think this line and 113 needs to be reverted to latest staging where `properties` is used on line 113 and `original_file` is used on line 116. Unless this change affects the audit log then we need to discuss why the change is needed.